build/NuSpecs/UmbracoCms.Core.nuspec

```diff
@@ -28,17 +28,17 @@
       <dependency id="MySql.Data" version="[6.9.8, 7.0.0)" />
       <dependency id="xmlrpcnet" version="[2.5.0, 3.0.0)" />
       <dependency id="ClientDependency" version="[1.9.2, 2.0.0)" />
-      <dependency id="ClientDependency-Mvc5" version="[1.8.0, 2.0.0)" />
+      <dependency id="ClientDependency-Mvc5" version="[1.8.0.0, 2.0.0)" />

```

Comment:
@Shazwazza Not sure.. that's how it's listed in csproj and packages.config: https://github.com/umbraco/Umbraco-CMS/commit/e1a93e9b3f9458ca7cd1c28ccc189ca95b7a174b